cql3/selection/selection.hh

```diff
@@ -279,7 +279,7 @@ public:
         }
         void reset(const partition_key* = nullptr) {
         }
-        uint32_t get_rows_dropped() const {
+        uint64_t get_rows_dropped() const {
```

Comment:
Nit: auto maybe?

Reply:
Will this work here? Or do you mean to change return 0 to return 0ull too?

Reply:
I guess it's ok to just leave it as is for now.